### ISSUE TITLE
Gerber File

### ISSUE THREAD

**Issue by Stewie-040882** [ProfBoots/Mini_Excavator#1]
Hi Pro-Boots,
   Love your work, do i need to join your page to have access to the gerber file? 
Thanks

**Comment by omtanmay** [comment]
Did you get it???

**Comment by Stewie-040882** [comment]
no, im assuming hes too busy with his next project.. i even asked on youtube but no answers. on his page he references the gerber file, so i assume you need to pay the subscription. sorry for the late reply, but i checked my junk mail and it had gone there. 
   i have some pcb prototyping board and pins, so gonna attempt to make one that way, while i wait for his next project

**Comment by omtanmay** [comment]
> no, im assuming hes too busy with his next project.. i even asked on youtube but no answers. on his page he references the gerber file, so i assume you need to pay the subscription. sorry for the late reply, but i checked my junk mail and it had gone there. 
>    i have some pcb prototyping board and pins, so gonna attempt to make one that way, while i wait for his next project 

Yes I saw his website u have to take his cource for the gerber
